INFO: 2020-03-17 21:25:45 
-------- START EXPERIMENT --------
INFO: 2020-03-17 21:25:45 
- PROPERTIES -
INFO: 2020-03-17 21:25:45 
{'data': 'elliptic',
 'data_config_file': 'data_config.yaml',
 'elliptic_args': {'encode_classes': True,
                   'feat_sets': ['LF'],
                   'inc_unknown': False,
                   'indexed_by_time': True,
                   'train_size': 0.7},
 'evaluation_metrics': ['precision', 'recall', 'f1', 'f1_micro', 'confusion'],
 'log_path': 'logs/',
 'models': ['xg_boost'],
 'save_log': True,
 'xg_boost_args': {'arguments': {'n_jobs': 4, 'verbosity': 1},
                   'extract_results': {'evaluation_metrics': ['precision',
                                                              'recall',
                                                              'f1',
                                                              'f1_micro',
                                                              'confusion',
                                                              'feature_importance'],
                                       'result_type': 'deterministic'},
                   'persist_props': {'method': 'save',
                                     'save_path': 'saved_models/experiment_1/tuned'}}}
INFO: 2020-03-17 21:25:45 
--- START BUILDING DATASET ---
INFO: 2020-03-17 21:25:45 
- DATASET -
INFO: 2020-03-17 21:25:45 
elliptic
INFO: 2020-03-17 21:25:45 
- CONFIG -
INFO: 2020-03-17 21:25:45 
'data_config.yaml'
ERROR: 2020-03-17 21:25:45 
(<class 'FileNotFoundError'>, FileNotFoundError(2, 'No such file or directory'), <traceback object at 0x7fa668b61820>)
Traceback (most recent call last):
  File "experiment.py", line 195, in <module>
    dataset = build_dataset(args)
  File "experiment.py", line 36, in build_dataset
    encode_classes=elliptic_args.encode_classes)
  File "/home/delinvas/repos/aml-crypto-graph/src/cryptoaml/datareader/data.py", line 26, in get_data
    with open(config_file, 'r') as stream:
FileNotFoundError: [Errno 2] No such file or directory: 'data_config.yaml'
